--- node_modules/hexo-generator-index/lib/generator.js.orig	2018-02-06 14:33:51.563128048 +0100
+++ node_modules/hexo-generator-index/lib/generator.js	2018-02-06 14:35:08.733344121 +0100
@@ -4,7 +4,7 @@
 
 module.exports = function(locals) {
   var config = this.config;
-  var posts = locals.posts.sort(config.index_generator.order_by);
+  var posts = locals.posts.filter(function(post) { return post.index !== false ; }).sort(config.index_generator.order_by);
   var paginationDir = config.pagination_dir || 'page';
   var path = config.index_generator.path || '';
 
